calico_upgrade/pkg/commands/constants/help.go

```diff
@@ -51,12 +51,12 @@ environments are all prefixed with "V1_"
 
     ` + FileValidationErrors + `
       This contains a list of errors that occurred when validating the v3
-      resources that were otherwise successfully converted from v1.  These
+      resources that were otherwise successfully converted from v1. These
       errors may indicate an issue such as a migrated name that is too long
       and would need resolving by deleting/re-adding the resource with a
-      shorter name before attempting the upgrade.  Or these errors may
-      indicate an issue with the migration script itself.  If you believe the
-      script is at fault, it is recommended to raise a GitHub issue at
+      shorter name before attempting the upgrade. Or these errors may
+      indicate an issue with the migration script itself. If you believe the
+      script is at fault, please raise a GitHub issue at
```

Comment:
I wonder if we should just have an "upgrade" channel in the CU slack and direct people to that.

Reply:
This is a really good idea since it would also let us know get a glimpse into how upgrades are going (and some sort of estimate on the number of users upgrading).

Reply:
it also avoids a massive flurry of duplicate issues or non-issues

Reply:
who has permission to create channels?